spec/models/account_result_spec.rb

```diff
@@ -416,7 +385,7 @@
       
         date_updated: "TS",
       
-        revision: 1,
+revision: 1,
```

Comment:
Formatting is off in this file

Reply:
Not according to this file link: https://github.com/FreeClimbAPI/ruby-sdk/blob/6d3a3f8a6dd5d1266cf56795ea0fb3dfe478aceb/spec/models/account_result_spec.rb

Reply:
See screenshot
<img width="533" alt="image" src="https://github.com/user-attachments/assets/2dc2368d-5090-452d-9aa4-7426cec441c6">